my-ansible-playbook.yml

```diff
@@ -0,0 +1,73 @@
+- name: Install To Do App on new web server
+  hosts: webservers
+  remote_user: ec2-user
+
+  vars_prompt:
+    - name: trello_api_key
+      prompt: Trello API Key
+
+    - name: trello_api_token
+      prompt: Trello API Token
+
+    - name: trello_board_id
+      prompt: Trello Board ID
+
+  tasks:
+  - name: Install Git
+    ansible.builtin.yum:
+      name: git
+      state: present
+    become: yes
+
+  - name: Install Python3
+    ansible.builtin.yum:
+      name: python3
+      state: present
```

Comment:
I think this task ought to have `become: yes` as well, like the "install git" task.

Reply:
Added become: yes as suggested